src/components/media-loader.js

```diff
@@ -589,7 +589,7 @@ AFRAME.registerComponent("media-loader", {
           { once: true }
         );
         this.el.setAttribute("floaty-object", { reduceAngularFloat: true, releaseGravity: -1 });
-        let batch = !disableBatching;
+        let batch = !disableBatching && forceImageBatching;
```

Comment:
nit : These flags are awkward. Batch if it's not disabled and it's forced on... but forceImageBatching can be disabled...

Reply:
Yep, agreed super awkward... The way it works right now is batching is not enabled by default, it becomes enabled if you either `forceImageBatching` or `forceMeshBatching`, the `disableBatching` flag no longer really makes too much sense since its off by default.. The intent was to leave things in the state they would be in for if/when we enable batching again by default... It could be awhile so maybe warrants revisiting that..